src/DevAdventCalendarCompetition/DevAdventCalendarCompetition.Repository/AdminRepository.cs

```diff
@@ -32,46 +32,43 @@ public void UpdateDates(Test test)
                 throw new System.ArgumentNullException(nameof(test));
             }
 
-            var dbTest = this.dbContext.Set<Test>().FirstOrDefault(el => el.Id == test.Id);
+            var dbTest = this._dbContext.Set<Test>().FirstOrDefault(el => el.Id == test.Id);
             if (dbTest != null)
             {
                 dbTest.StartDate = test.StartDate;
                 dbTest.EndDate = test.EndDate;
-                this.dbContext.SaveChanges();
+                this._dbContext.SaveChanges();
             }
         }
 
         public void UpdateEndDate(Test test)
         {
-            if (test is null)
-            {
-                throw new System.ArgumentNullException(nameof(test));
-            }
-
-            var dbTest = this.dbContext.Set<Test>().FirstOrDefault(el => el.Id == test.Id);
+            var dbTest = this._dbContext.Set<Test>().FirstOrDefault(el => el.Id == test.Id);
             if (dbTest != null)
             {
+#pragma warning disable CA1062 // Validate arguments of public methods
```

Comment:
please add in the line number 46 null check for object test and remove pragma warning